portal/static/less/exerciseDiet.less

```diff
@@ -0,0 +1,3556 @@
+@mainBackgroundColor: #304040;
+@lightBlueBackgroundColor:  #4a6161;
+@lightGreenBackgroundColor: #44504f;
+@HRColor: #ececec;
+@alternativeBackgroundColor: #253534;
+@bodyTextColor: #4a4a4a;
+@navbarBorderColor: #d7dde0;
+@navbarColor: darken(#7c959e, 5%);
+@navbarBackgroundColor: rgba(210,209,197,.3);
+@highlightColor: #f7ddbd;
+@darkenHighlightColor: #625748;
+@modalHighlightColor: #6a5944;
+@headerColor: #FFF;
+@tnthPrimaryColor: #7C959D;
+@btnTruenthPrimaryColor: #7C959D;
+@btnTruenthArrowColor: #4c5e6e;
+@btnTruenthPrimaryHoverColor: #576E76;
+@miniBoxContainerBackgroundColor: #5f676e;
+@defaultBoxBackgroundColor: rgba(51,54,56,.8);
+@miniBoxBackgroundColor: rgba(51,54,56,.8);
+@miniBoxHeaderColor: #dccab2;
+@miniBoxHeaderFontSize: 18px;
+@miniBoxHeaderFontWeight: 500;
+@iceboxSubheadColor: #a9b2b7;
+@cellBorderColor: #959eab;
+@cellCaptionHeaderColor: #5a6d75;
+@cellHeaderColor: #5a6d75;
+@modalCellBackgroundColor: hsla(113, 16%, 91%, 0.4);
+@modalCellBorderColor: #caced4;
+@cellBackgroundColor: #697e86;
+@cellHoverColor: darken(#697e86, 10%);
+@sectionBorderColor: #343;
+@sectionBorderColor: #3b4e4e;
+@modalHeaderBackgroundColor: #607277;
+@modalHeadingColor: #4b5e6f;
+@modalSubHeadingColor: #4a4c4d;
+@modalHeadingBorderColor: #cfd5d8;
+@inputBackgroundColor: #f1efec;
+@resourceLinkColor: #bedfea;
+@resourceBackgroundColor: rgba(59,68,73,.5);
+@infoButtonBackgroundColor: #8f9385;
+@figureBackgroundColor: #4c5e6e;
+@IntroOverlayBackgroundColor: #333;
+@ExerciseOverlayBackgroundColor: #313535;
+@RecipieOverlayBackgroundColor: #323535;
+@calloutTextColor: #FFF;
+@calloutTextHoverColor: #ccd7dd;
+@mobileMainNavFontSize: 12px;
+@mainNavFontSize: 13px;
+@mobileMainNavCellHeight: 52px;
+@mainNavCellHeight: 60px;
+@contentFontSize: 24px;
+@contentSubheadingFontSize: 32px;
+@mobileSubheadingFontSize: 24px;
+@contentPromptFontSize: 30px;
+@mobileContentFontSize: 20px;
+@buttonFontSize: 18px;
+@startButtonFontSize: 19px;
+@tableCellFontSize: 17px;
+@mobileTableCellFontSize: 14px;
+@mobileTableCellFontSize: 16px;
+@smallerTextFontSize: 16px;
+@mobileButtonFontSize: 16px;
+@miniBoxTitleFontSize: 16px;
+@miniBoxFontSize: 18px; //
+@modalBodyFontSize: 18px;
+@modalHeadingSize: 24px;
+@heroTitleFontSize: 38px;
+@h2FontSize: 37px;
+@h2MobileFontSize: 30px;
+@h3FontSize: 30px;
+@h3MobileFontSize: 26px;
+@h4FontSize: 22px;
+@h4MobileFontSize: 18px;
+@h6FontSize: 15px;
+@h6MobileFontSize: 12px;
+@referenceFontSize: 18px;
+
+@bodyFontFamily: "akzidenz_grotesk_medium", Arial, "Helvetica Neue", Helvetica, sans-serif;
+@modalHeaderFontWeight: 700;
+
+body {
+    margin: 0;
+    padding: 0;
+    line-height: 1.35em;
+    letter-spacing: .02em;
+    font-weight: 500;
+    font-style: normal;
+    color: #333;
+    background-color: #FFF;
+    font-family: @bodyFontFamily;
+}
+
+:not(.is-scrolled) .theme--intro-light .logo {
+    color: black;
+    background-image: url(../../../../gil/static/img/logo-base.png);
```

Comment:
Are you able to use an absolute path here?

Reply:
noting here that this is outdated css and I have since removed it.